mailipy/send.py

```diff
@@ -39,7 +41,12 @@ def main():
         sys.exit(1)
 
     print("You are about to send %d emails." % len(emails))
-    password = getpass.getpass("Password for %s@%s: " % (args.username, args.server))
+    if args.password is not None:
+        password = args.password
+    elif args.password_stdin:
```

Comment:
I don't really like that if both `--password` and `--password-stdin` are provided, we silently use the former without warning / erroring.

Reply:
Fair, we can make them mutually exclusive.